client/main.lua

```diff
@@ -65,7 +65,7 @@ local keybind = lib.addKeybind({
 })
 local scaleform = nil
 lib.callback.register('qbx_binoculars:client:toggle', function()
-    if cache.vehicle or QBX.PlayerData.metadata.isdead or QBX.PlayerData.metadata.ishandcuffed or QBX.PlayerData.metadata.inlaststand then return end
+    if cache.vehicle or or IsPedSwimming(cache.ped) or QBX.PlayerData.metadata.isdead or QBX.PlayerData.metadata.ishandcuffed or QBX.PlayerData.metadata.inlaststand then return end
```

Comment:
You have 'or' two times here

Reply:
Yes I know I'm fixing it along with something else rn